Standards/scs-0103-v1-standard-flavors.md

```diff
@@ -0,0 +1,145 @@
+---
+title: SCS Standard Flavors and Properties
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+## Motivation
+
+In OpenStack environments there is a need to define different flavors for instances.
+The flavors are pre-defined by the operator, the customer can not change these.
+OpenStack providers thus typically offer a large selection of flavors.
+
+While flavors can be discovered (`openstack flavor list`), it is helpful for users (DevOps teams),
+to have a guaranteed set of flavors available on all SCS clouds, so these need not be discovered.
+
+## Properties (extra specs)
+
+The following extra specs are recognized, together with the respective semantics:
+
+- `scs:name-vN=NAME` (where `N` is `1` or `2`, and `NAME` is some string) means that the
+  flavor is one of the
+  standard SCS flavors, and the requirements of Section "Standard SCS flavors" below apply.
+- `scs:cpu-type=shared-core` means that _at least 20% of a core in >99% of the time_,
```

Comment:
Don't we need to provide the full list of allowable and standardized options here?
I.e. `crowded-core` (corresponding to an `L` encoded in the name), `shared-core` (`V), `dedicated-thread` (`T`), or `dedicated-core` (`C`)?

Reply:
This standard is about standard flavors. Therefore, I only listed the values that we need for the standard flavors.

Reply:
OK, we discussed this in the Team IAM call and I pushed https://github.com/SovereignCloudStack/standards/pull/319/commits/074b12a7102c74a47db74609456a3389c5348589 as a consequence.
Rationale: Without having the full extra_specs discoverability/transparency thing defined, we have introduced the extra_specs `scs:name-v1`, `scs:name-v2`, `scs:cpu-type` and `scs:diskN-type` here on the fly. If we do so, let's cover them completely and leave all other extra_specs in a future standard. This avoids confusion for the reader and avoids providers doing wild things that are hard to change once they are out in the wild.

Reply:
I don't disagree, but I strongly oppose the breaking of due process here. It takes you a full week to write your comment, and then it can't happen quickly enough, so you push the change yourself. This is very disturbing.

Reply:
I wouldn't have minded a direct contribution if the back and forth hadn't been as sluggish. I had a feeling of being left out, no real cooperation happening, and your commit to the branch was only adding to that feeling.